lib/qwerty/crop.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'draw'
+module Qwerty
+  class Draw
+    class Crop < Ruote::Participant
+      def on_workitem
+        workitem.fields['draw']['crop'] = {
+          :clip => 'rect(0px,60px,200px,0px);'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.